### PR TITLE
SAK-48298 Manage Tools - Customize tool instances page: x for removal should be red

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-48298

Waiting for Andrea to take a look at the changes, Until this pr will be in the draft. Thank you :)

Andrea Approved, Ready for review.

### CODE REVIEW

File: library/src/skins/default/src/sass/modules/tool/sitemanage/_sitemanage.scss [hunk 1]
```diff
@@ -426,7 +426,7 @@
   }
   .removeTool {
     text-decoration: none !important;
-    color: var(--errorBanner-color);
+    color: var(--errorBanner-color) !important;
```
Comment (adrianfish) [inline library/src/skins/default/src/sass/modules/tool/sitemanage/_sitemanage.scss:429]: Why do you need the !important? What is overriding it?

Comment (kunaljaykam) [reply library/src/skins/default/src/sass/modules/tool/sitemanage/_sitemanage.scss:429]: <a> is overriding

Comment (adrianfish) [reply library/src/skins/default/src/sass/modules/tool/sitemanage/_sitemanage.scss:429]: What scss file is that coming from? Is there no way the order can be corrected?

**Comment by kunaljaykam** [comment]
Another pr https://github.com/sakaiproject/sakai/pull/11298

**Pull request closed without merging**